diff --git a/zshrc b/zshrc
index e728530..2bf04b0 100644
--- a/zshrc
+++ b/zshrc
@@ -72,7 +72,7 @@ alias grep='grep -i --color=auto --exclude-dir={.bzr,CVS,.git,.hg,.svn}'
 #color cat
 alias ccat='pygmentize -O style=borland -f console256 -g'
 
-export PATH=$HOME/dotfiles-oschina/do:$HOME/opt/bin:$HOME/opt/sbin:$HOME/Library/Python/3.6/bin:$PATH
+export PATH=$HOME/dotfiles-oschina/do:$HOME/opt/bin:$HOME/opt/sbin:$PATH
 
 # find shorthand
 function f () {
@@ -189,3 +189,16 @@ test -e "${HOME}/.iterm2_shell_integration.zsh" && source "${HOME}/.iterm2_shell
 export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
 [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+
+
+. "$HOME/.local/bin/env"
+
+# Added by Windsurf
+export PATH="/Users/z/.codeium/windsurf/bin:$PATH"
+
+# Added by Antigravity
+export PATH="/Users/z/.antigravity/antigravity/bin:$PATH"
+
+# opencode
+export PATH=/Users/z/.opencode/bin:$PATH
+export PATH="/opt/homebrew/opt/openjdk@17/bin:$PATH"
